code/datums/status_effects/debuffs.dm

```diff
@@ -155,7 +156,11 @@
 /datum/status_effect/electrode/tick()
 	var/diff = world.time - last_tick
 	if(owner)
-		owner.adjustStaminaLoss(max(0, stamdmg_per_ds * diff)) //if you really want to try to stamcrit someone with a taser alone, you can, but it'll take time and good timing.
+		var/mob/living/carbon/C = owner
+		if(HAS_TRAIT(C, TRAIT_TASED_RESISTANCE))
```

Comment:
not the best way to do this but since it ticks every second it isn't that horrible i guess..
actually wait yeah it slows down, it's much better to just check it on apply and have the trait add wipe any tased effect when it's added.

Reply:
Alright cool beans.